### PR TITLE
Renaming application.tailwind.css to tailwind.css

### PR DESCRIPTION
**Pull request by elhalvers**
## Getting Tailwind CSS to work

After running bundle and yarn and db:create, db:migrate, db:seed. I got an error when trying to run the server. I did a little research and then renamed 'assets/stylesheets/ application.tailwind.css to /tailwind.css and then I was able to see the basic page populated with the 'seeded users'

### CODE REVIEW

**Comment by adrianvalenz** [comment]
Hey @elhalvers ! How is it going? I was wondering if you had a snapshot of that weird error you got? I was trying to reproduce the error locally and I didn't see anything that would cause Tailwind to not work (it's working fine as I can see).

Why did you run `yarn` in the project? This project was created via the default new command with no JavaScript bundling, meaning `-j esbuild` or the like was not passed so it defaults to using import maps...and from what I understand when the application was created and the `--css tailwind` flagged was passed it uses the `tailwindcss-rails` gem to get Tailwind to work [without node or NPM required](https://tailwindcss.com/blog/standalone-cli).

I do see there was a problem with the seed file not populating but I think it was due to how it was set up.

Thank you for reading my comment and your time!

Adrian 😊👍

**Comment by elhalvers** [comment]
Hi Adrian,

Thanks for reaching out! I can try again from scratch and see if it works.
I think I am just in the habit of running bundle and yarn when I clone a
new repo.

Let me try again. I will follow the ReadMe file as is and see if it is
sufficient or if some updates would be helpful.

Thank you Adrian!

: )

Eric
